recipes/backports.unittest_mock/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.3" %}
```

Comment:
In the meantime version `1.4` has been released, so we can update to that version.